ci/job.yml

```diff
@@ -113,15 +113,79 @@
             echo ${sha1}
             echo ${GIT_URL}
             echo ${GIT_BRANCH}
+            CI_DEBUG=0
             if [ ${ghprbTargetBranch} == "openshift" ]
             then
               git rebase origin/${ghprbTargetBranch}
               cd ci
-              sh trigger_ci.sh ${GIT_URL} ${GIT_BRANCH} ${sha1}
+              sh trigger_ci.sh ${GIT_URL} ${GIT_BRANCH} ${sha1} ${CI_DEBUG}
             else
               echo "PR's target branch is not openshift"
             fi
 
+
+- job:
+    name: 'centos-container-pipeline-service-ci-pr-functional-debug'
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    ci_project: container
+    properties:
+        - github:
+            url: https://github.com/CentOS/container-pipeline-service/
```

Comment:
We can have git URL specified in same way throughout the job like
```
            url: https://github.com/CentOS/container-pipeline-service.git
```
specified [here](https://github.com/CentOS/container-pipeline-service/pull/634/files#diff-091765ca1f913cb1fc03c4831eb431bbR166) at line no 165 later in the file.